### PR TITLE
Google Analytics

### PR DESCRIPTION
**Pull request by laurence-hudson-tessella**

### CODE REVIEW

File: src/PublicSite/web/js/analytics.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/* Google Universal Analytics tracking code with the the ABRAID www.abraid.ox.ac.uk account ID,
```
Comment (helena-patching-tessella) [inline src/PublicSite/web/js/analytics.js]: "the the ABRAID"

**Comment by edwiles** [comment]
LGTM

File: src/PublicSite/web/js/app/atlas/AtlasView.js [hunk 2]
```diff
@@ -42,6 +42,9 @@ define([
                     wmsLayerParameterFactory.createLayerParametersForDisplay(layer));
 
                 self.map.addLayer(self.currentLayer);
+
+                ko.postbox.publish(
+                    "tracking-action-event", { "category": "atlas", "action": "layer-view", "label": layer });
```
Comment (helena-patching-tessella) [inline src/PublicSite/web/js/app/atlas/AtlasView.js]: new line for ); or all on one line?

**Comment by helena-patching-tessella** [comment]
LGTM

**Pull request merged**